parser/src/keys/mod.rs

```diff
@@ -5,19 +5,45 @@ use once_cell::sync::Lazy;
 use parking_lot::Mutex;
 use rustc_hash::FxHashMap as HashMap;
 
-#[cfg(any(target_os = "linux", target_os = "unknown"))]
 mod linux;
-#[cfg(any(target_os = "linux", target_os = "unknown"))]
-pub use linux::*;
-
-#[cfg(target_os = "windows")]
 mod windows;
-#[cfg(target_os = "windows")]
-pub use windows::*;
 
 mod mappings;
 pub use mappings::*;
 
+#[derive(Clone, Copy)]
+pub enum Platform {
+    Win,
+    Linux,
+}
+
+pub static OSCODE_MAPPING_VARIANT: Mutex<Platform> = Mutex::new({
```

Comment:
I'm pretty sure there are better options to use here than `Mutex`. I tried `once_cell::sync::Lazy` but couldn't get it to work.

Reply:
If this is intended to be mutable, then this does need to be a mutex. Otherwise, it should be able to be const computed, though I don't play with const very much. Probably no need for Lazy either way.

Reply:
Or I suppose instead of an enum it can be an AtomicU8 wrapper type

Reply:
Yes I want it to be be mutable and settable from outside of crate. Mutex shouldn't be much overhead after processing loop starts, I've checked where conversion would occur in the conversation below.

Reply:
Shouldn't matter after dbb4163